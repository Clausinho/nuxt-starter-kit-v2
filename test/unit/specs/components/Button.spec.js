import { shallow } from '@vue/test-utils';
import { createRenderer } from 'vue-server-renderer';
import Component from '~/components/Button.vue';

describe('Component Button', () => {
  it('renders to a snapshot', () => {
    const renderer = createRenderer();
    const wrapper = shallow(Component, {});
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err);
      expect(str).toMatchSnapshot();
    });
  });
});
