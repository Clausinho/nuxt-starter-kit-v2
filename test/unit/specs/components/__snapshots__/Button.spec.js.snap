// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Button renders to a snapshot 1`] = `<button class="button" style="color:#42b983;border-color:#42b983;">!</button>`;
